backend/init_db.py

```
@@ -0,0 +1,28 @@
+    print("既存のテーブルを削除しています...")
+    
+    # 既存のテーブルをCASCADEで削除
+    with engine.connect() as conn:
+        existing_tables = ['user_items', 'user_skins', 'spots', 'users', 'skins']
+        
+        for table_name in existing_tables:
+            try:
+                print(f"  削除中: {table_name}")
+                conn.execute(text(f"DROP TABLE IF EXISTS {table_name} CASCADE"))
+                conn.commit()
+            except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)